Release build fails

I get the following output from `zig build -Doptimize=ReleaseSafe` (and any other release mode; x86_64 linux; zig version is `0.12.0-dev.293+f33bb0228`):

```
zig build-exe arocc ReleaseSafe native: error: the following command failed with 1 compilation errors:
/home/ehaas/.local/zig-linux-x86_64-0.12.0-dev.293+f33bb0228/zig build-exe /home/ehaas/source/arocc/src/main.zig -lc -OReleaseSafe --cache-dir /home/ehaas/source/arocc/zig-cache --global-cache-dir /home/ehaas/.cache/zig --name arocc --mod system_defaults::/home/ehaas/source/arocc/zig-cache/c/c8adec6bc0f38ebe22f49db72073261e/options.zig --mod build_options::/home/ehaas/source/arocc/zig-cache/c/cdb917547a33511d1b8c3c9d78f3a7b6/options.zig --mod zig::/home/ehaas/source/arocc/deps/zig/lib.zig --deps build_options,system_defaults,zig --listen=-
Build Summary: 2/5 steps succeeded; 1 failed (disable with --summary none)
install transitive failure
└─ install arocc transitive failure
   └─ zig build-exe arocc ReleaseSafe native 1 errors
src/Driver.zig:481:91: error: switch must handle all possibilities
        d.processSource(tc, d.inputs.items[0], builtin, user_macros, fast_exit) catch |e| switch (e) {
                                                                                          ^~~~~~
src/Driver.zig:481:91: note: unhandled error value: 'error.TooManyMultilibs'
referenced by:
    main: src/main.zig:67:11
    callMain: /home/ehaas/.local/zig-linux-x86_64-0.12.0-dev.293+f33bb0228/lib/std/start.zig:571:24
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
```

Debug builds work as expected. The cause is that `error.TooManyMultilibs` is missing in the relevant switch prongs in Driver.zig, but it's not clear to me why debug builds would work. Is this a Zig bug or is something else going on there?

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -25,6 +25,10 @@ jobs:
         with:
           version: master
 
+      - name: Fmt
+        run: zig fmt . --check
+        if: matrix.os == 'ubuntu-latest'
+
       - name: Build
         run: zig build
 
@@ -32,9 +36,8 @@ jobs:
         run: zig build -Dtarget=arm-linux
         if: matrix.os == 'ubuntu-latest'
 
-      - name: Fmt
-        run: zig fmt . --check
-        if: matrix.os == 'ubuntu-latest'
+      - name: Build release
+        run: zig build -Doptimize=ReleaseSafe
 
       - name: Run Tests
         run: zig build test

--- a/src/Driver.zig
+++ b/src/Driver.zig
@@ -479,21 +479,21 @@ pub fn main(d: *Driver, tc: *Toolchain, args: []const []const u8) !void {
 
     if (fast_exit and d.inputs.items.len == 1) {
         d.processSource(tc, d.inputs.items[0], builtin, user_macros, fast_exit) catch |e| switch (e) {
-            error.OutOfMemory => return error.OutOfMemory,
             error.FatalError => {
                 d.comp.renderErrors();
                 d.exitWithCleanup(1);
             },
+            else => |er| return er,
         };
         unreachable;
     }
 
     for (d.inputs.items) |source| {
         d.processSource(tc, source, builtin, user_macros, fast_exit) catch |e| switch (e) {
-            error.OutOfMemory => return error.OutOfMemory,
             error.FatalError => {
                 d.comp.renderErrors();
             },
+            else => |er| return er,
         };
     }
     if (d.comp.diag.errors != 0) {

